src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Helper/TranslationHelper.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Microsoft.PowerToys.Run.Plugin.WindowsSettings.Properties;
+using Wox.Plugin.Logger;
+
+namespace Microsoft.PowerToys.Run.Plugin.WindowsSettings.Helper
+{
+    /// <summary>
+    /// Helper class to easier work with translations.
+    /// </summary>
+    internal static class TranslationHelper
+    {
+        /// <summary>
+        /// Translate all settings of the given list with <see cref="WindowsSetting"/>.
+        /// </summary>
+        /// <param name="settingsList">The list that contains <see cref="WindowsSetting"/> to translate.</param>
+        internal static void TranslateAllSettings(in IEnumerable<WindowsSetting>? settingsList)
+        {
+            if (settingsList is null)
+            {
+                return;
+            }
+
+            foreach (var settings in settingsList)
+            {
+                var area = Resources.ResourceManager.GetString($"Area{settings.Area}");
+                var name = Resources.ResourceManager.GetString(settings.Name);
+                var type = Resources.ResourceManager.GetString(settings.Type);
```

Comment:
@mykhailopylyp, @TobiasSekan 
I'd like aligned resx string naming.

Property|Now|I think about
------------ |------------ | -------------
App|SettingsApp|AppSetingsApp
Area|AdminTools|AreaAdminTools
Name (alt names and names)|AdminTools|NameAdminTools 
Note|NoteIntruducedIn19043|NoteIntruducedIn19043

Questions:
- Should we add the prefix by code or should we write it into json? What do you prefer?
- Should we have a dot or underline between prefix and string (e.g. `App_SettingsApp`)?

Reply:
The cells `Now` and `I think about` in the last row in your table is the same.

Reply:
`AreaAdminTools` is not present in resx file

Reply:
I have add your suggestion from the first row

Reply:
> `AreaAdminTools` is not present in resx file

I know. But maybe in the feature.

Reply:
> I have add your suggestion from the first row

This was only an example and creates more differences in naming.

Let's merge. I will create a tracking issue later for all suggestions.